data/souhaibsfaxi.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "Souhaib Sfaxi",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Open Source Enthusiast!",
+  "avatar": "https://github.com/souhaibsfaxi.png",
+  "tags": ["javascript", "typescript"],
+  "socials": [
+    
+    { "platform": "github", "url": "https://github.com/souhaibsfaxi" }
```

Should be `icon`, AFAIK.
```suggestion
    { "icon": "github", "url": "https://github.com/souhaibsfaxi" }
```